test.sh

```diff
@@ -300,4 +301,10 @@ if [[ "$RUN" =~ "godep-restore" ]] ; then
   end_context #godep-restore
 fi
 
+if [[ "$RUN" =~ "errcheck" ]] ; then
+  start_context "errcheck"
+  run_and_comment errcheck -ignore 'io:Write,Close,os:Remove,net/http:Write,github.com/letsencrypt/boulder/log:.*,github.com/letsencrypt/boulder/metrics:.*' -ignorepkg 'github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/cactus/go-statsd-client/statsd,github.com/letsencrypt/boulder/rpc/pb' ./akamai/ ./bdns/ ./bin/ ./ca/ ./cmd/ ./core/ ./data/ ./log/ ./mail ./metrics ./mocks ./policy ./probs ./publisher ./ra ./reloader ./rpc ./sa ./test ./va ./wfe
```

Comment:
I think you might be able to remove `log` from this ignore list since I refactored that package to not return errors.